bower.json

```diff
@@ -9,15 +9,15 @@
     "output"
   ],
   "dependencies": {
-    "purescript-prelude": "^4.0.0",
-    "purescript-aff": "^5.0.0",
-    "purescript-random": "^4.0.0",
-    "purescript-quickcheck": "^5.0.0",
-    "purescript-spec": "^3.1.0"
+    "purescript-prelude": "^5.0.0",
+    "purescript-aff": "^6.0.0",
+    "purescript-random": "^5.0.0",
+    "purescript-quickcheck": "master",
```

Comment:
I see this repo has [a "prepare 7.0.0 release" commit](https://github.com/purescript/purescript-quickcheck/commit/7db589bc9a5fb058291863900c5d93c03cc28e5c) with all the right updates, but no actual version tag.

@thomashoneyman @kl0tl

Reply:
There’s a v7 tag:
https://github.com/purescript/purescript-quickcheck/releases

Reply:
Oh, I must have missed that. My bad. Fixed now.